WebRoot/js/zimbraMail/calendar/view/ZmCalDayView.js

```diff
@@ -327,6 +334,13 @@ function(abook) {
 	this._scrollToTime(8);
 };
 
+ZmCalDayView.__onMouseWheel =
+function(elementId, event) {
+	// stop smooth scrolling
+	event.preventDefault();
+	document.getElementById(elementId).scrollTop -= event.wheelDelta / 2;
```

Comment:
I am not very sure if browsers optimise this behaviour and not call document.getElementById twice for evaluations
better to get dom element in a variable and then perform operation like below
```
var elm = document.getElementById(elementId);
elm.scrollTop -= event.wheelDelta / 2;
```

Reply:
fixed it